src/components/authentication/mfa/SetupTOTP.tsx

```diff
@@ -31,12 +31,10 @@ const SetupTOTP = ({ user, mfaStatus, incrementStep }: ISetupMFA) => {
     const getUser = async () => {
       try {
         const totpCode: string = await Auth.setupTOTP(user);
-        const authCode: string =
-          "otpauth://totp/AWSCognito:" +
-          user.getUsername() +
-          "?secret=" +
-          totpCode +
-          "&issuer=AWSCognito";
+        const name = "NHS Trainee Self-Service";
```

Comment:
If it's only used in one place, is there any value in defining it as a a `const`?  The convention in Java would be to inline the value.

Reply:
Fair point. I was using twice, as issuer query string parameter as well. However, since this doesn't appear necessary I shall pass inline.